docker-for-mac/mutagen-caching.md

```diff
@@ -0,0 +1,111 @@
+---
+description: High-performance two-way file sync for volume mounts
+keywords: mac, mutagen, volumes, file sync,
+title: Mutagen-based caching
+---
+
+Docker Desktop for Mac on Edge has a new file sharing feature which performs
+a continuous two-way sync of files between the host and containers using
+[Mutagen](https://mutagen.io/). This feature is ideal for app development where:
+
+- the source code tree is quite large
+- the source is edited on Mac
+- the source is compiled and run interactively inside Linux containers.
+
+The following section contains an example to show how the Mutagen feature should be used to sync files between the host and containers .
```

Comment:
Nit: extra space before full stop.

Reply:
I think it would be useful to say that there are best practices later in the document. Something like:

This page contains an example to show how the Mutagen feature should be used to sync files between the host and containers. It also gives some best practices to maximize performance.

Reply:
Updated.